spec/features/form/tasklist_page_spec.rb

```diff
@@ -27,6 +27,16 @@
       managing_organisation: user.organisation,
     )
   end
+  let(:setup_completed_log) do
+    FactoryBot.create(
+      :case_log,
+      owning_organisation: user.organisation,
+      managing_organisation: user.organisation,
+      renewal: 1,
+      rent_type: 1,
+      startdate: Time.zone.local(2021, 5, 1),
+    )
+  end
```

Comment:
This is what the `:about_completed` trait in the factory aimed to do . You could use that and possibly rename it to setup completed instead